json/src/error.rs

```diff
@@ -92,6 +102,9 @@ impl fmt::Debug for ErrorCode {
             ErrorCode::MissingField(ref field) => write!(f, "missing field \"{}\"", field),
             ErrorCode::TrailingCharacters => "trailing characters".fmt(f),
             ErrorCode::UnexpectedEndOfHexEscape => "unexpected end of hex escape".fmt(f),
+            ErrorCode::Length(ref len) => write!(f, "incorrect value length {}", len),
```

Comment:
These changes are already contained in PR #11 and are likely to conflict if that PR is merged.